<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryDialog</class>
 <widget class="QDialog" name="HistoryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>Training History</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter</set>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="historyListWidget"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Clear History</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #f44336; color: white; padding: 10px 20px; border: none; border-radius: 5px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="returnButton">
        <property name="text">
         <string>Return</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #333; color: white; padding: 10px 20px; border: none; border-radius: 5px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
